<launch>
  <node pkg="test_package" type="simulator" name="simulator" output="screen">
    <rosparam param="init_state"> [1.5,0,0,0,0,0,-3,0,-2,0.7071,0,0,0.7071]</rosparam>
    <param name="kite_params" value="/Users/plistov/EPFL/ROS/ros_catkin_ws/devel/lib/test_package/umx_radian2.yaml" />
  </node>
  <node pkg="test_package" type="kite_visualization_node" name="visualisation_node" output="screen">
  </node>
  <node pkg="test_package" type="control_proxy_node" name="control_proxy" />
  <node pkg="rosserial_python" type="serial_node.py" name="serial_node">
    <param name="port" value="/dev/cu.usbserial-AL022K69"/>
    <param name="baud" value="250000"/>
  </node>
</launch>
